modules/installation-launching-installer.adoc

```diff
@@ -301,26 +293,25 @@ https://ovirtlab.example.com/ovirt-engine/api
 ----
 endif::openshift-origin[]
 +
-.. For `Is the oVirt CA trusted locally?`, enter `Yes` since you have already set up a CA certificate. Otherwise, enter `No`.
-.. For `oVirt's CA bundle`, if you entered `Yes` for the preceding question, copy the certificate content from `/etc/pki/ca-trust/source/anchors/ca.pem` and paste it here. Then, press `Enter` twice. Otherwise, if you entered `No` for the preceding question, this question does not appear.
-.. For `oVirt engine username`, enter the user name and profile of the {rh-virtualization} administrator using this format:
+.. The installer automatically generates a CA certificate. For `Would you like to use the above certificate to connect to Engine?`, enter `Yes`. Otherwise, enter `No`. If you answer `No`, you need to install {product-title} in insecure mode.
```

Comment:
I think that this step could be improved. If I follow it exactly, I've already entered "Yes" since you've instructed me to do so, before I even see the "No" option.

I'd consider rewording it somehow to give the two options at the same time. Like "Enter `Yes` to <do this> or enter `No` to <do this>. And then add any caveats about selecting no. I also am unsure what you mean about installing in insecure mode. We're currently installing OCP right now, what exactly do I need to do then if I choose no?

Also, "you need to" -> "you must"

Reply:
Fixed.